Assets/MRTK/Providers/LeapMotion/Editor/LeapMotionDeviceManagerProfileInspector.cs

```diff
@@ -22,18 +23,35 @@ public class LeapMotionDeviceManagerProfileInspector : BaseMixedRealityToolkitCo
         protected LeapMotionDeviceManagerProfile instance;
         protected SerializedProperty leapControllerOrientation;
         protected SerializedProperty leapControllerOffset;
+
+        protected SerializedProperty leapVRDeviceOffsetMode;
+        protected SerializedProperty leapVRDeviceOffsetY;
+        protected SerializedProperty leapVRDeviceOffsetZ;
+        protected SerializedProperty leapVRDeviceOffsetTiltX;
+        protected SerializedProperty leapVRDeviceOrigin;
+
         protected SerializedProperty enterPinchDistance;
         protected SerializedProperty exitPinchDistance;
 
         private const string leapDocURL = "https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/CrossPlatform/LeapMotionMRTK.html";
 
+        // Used for setting the leapVRDeviceOrigin object reference value
+        Transform leapVRDeviceOriginTransform;
+
         protected override void OnEnable()
         {
             base.OnEnable();
 
             instance = (LeapMotionDeviceManagerProfile)target;
```

Comment:
Got this error for this line: 
```
InvalidCastException: Specified cast is not valid.
Microsoft.MixedReality.Toolkit.LeapMotion.Inspectors.LeapMotionDeviceManagerProfileInspector.OnEnable () (at Assets/MRTK/Providers/LeapMotion/Editor/LeapMotionDeviceManagerProfileInspector.cs:45)
```
Changing this line to `as LeapMotionDeviceManagerProfile` to avoid the InvalidCastException